modules/sdk-core/src/bitgo/tss/common.ts

```diff
@@ -95,7 +95,7 @@ export async function commonVerifyWalletSignature(params: {
     throw new Error('Invalid HSM GPG signature');
   }
 
-  const verificationResult = await walletSignature.verifyPrimaryUser([bitgoPub]);
+  const verificationResult = await walletSignature.verifyPrimaryUser([bitgoPub], null as unknown as undefined);
```

Comment:
```suggestion
  const verificationResult = await walletSignature.verifyPrimaryUser([bitgoPub], undefined);
```

confused on why we can't just pass in undefined. Also why this fixes the issue.

or alternatively change the types for `verifyPrimaryUser` to accept null as input?